packages/proxy/schema/secrets.ts

```diff
@@ -8,7 +8,7 @@ export const BaseMetadataSchema = z
         z.object({
           format: z.enum(["openai", "anthropic", "google"]),
           flavor: z.enum(["completion", "chat"]),
-          multimodal: z.boolean().nullish(),
+          multimodal: z.boolean().optional(),
```

Comment:
Why not use `nullish()`? It's generally safer (especially when writing data to a database)

Reply:
Let's ship!